app/helpers/products_helper.rb

```diff
@@ -2,4 +2,8 @@ module ProductsHelper
   def types_of_products
     Product.product_types.keys
   end
+
+  def product_calories(product_id)
+    ProductCalories.find_by(product_id: product_id)
```

Comment:
please use association instead of this method, as it generated additional query to db